### ISSUE TITLE
when is there going to be a new version that supports godot 4 and onward?

### ISSUE THREAD

**Issue by tummypics** [char-ptr/gdke#19]
i really hope one gets released soon or the tutorial gets updated because i've been trying to do it with the help of the other issues but i cant get it to work

**Comment by char-ptr** [comment]
yeah i know i've kinda neglected this recently. i really want to just get some time and get everything sorted out. i'm hoping to maybe do this over the next week but honestly not sure at the moment.

**Comment by tummypics** [comment]
it's alright life happens i get it i wish i could help but i dont know how to use rust in the slightest

**Comment by code-hasaki** [comment]
> yeah i know i've kinda neglected this recently. i really want to just get some time and get everything sorted out. i'm hoping to maybe do this over the next week but honestly not sure at the moment.是的，我知道我最近有点忽视了这一点。我真的想花点时间把一切都解决掉。我希望也许能在下周做到这一点，但老实说目前还不确定。

Please support 4.3. For example, this game.
https://pixeldrain.com/u/eEPCVwYG

**Comment by theultramage** [comment]
The guide is outdated; for 4.x, instead locate and xref the string "Can't open encrypted pack-referenced file" to find FileAccessPack::FileAccessPack(), a function similarly-shaped to what's in the guide. In particular, the way the global key gets loaded in is the same - a std::vector resized to 32 and then written in a `movzx xxx, byte ptr [i+keyptrreg]` loop. The number 32 (0x20), a vector debug string, and two global vars being loaded into registers in the same block makes it stand out in the disassembly pretty well.
